### DISCUSSION TITLE
Pass styled to custom props

### DISCUSSION THREAD

**Discussion by ryanahall** [nativewind/nativewind#138] [Q&A] [answered]
I'm using an external library called `react-native-floating-label-input` which has style props for various child components (e.g. `inputStyles` and `labelStyles`). Is it possible to pass the stylesheet generated directly to these props? Using `className` seems to only apply to the root element, resulting in some very strange styling.

Great library by the way, version 2 seems to be a big step forward.

**Comment by marklawlor** [comment, marked as answer]
You will need to wrap the component in `styled()` and tell it to transform these props. https://www.nativewind.dev/api/styled#styling-multiple-properties

The API changed for this in the latest version, so make sure you are on `2.0.0-next.53`

**Reply by ryanahall** [reply]
Thank you! This worked perfectly.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
